android/src/main/java/com/googlecast/GoogleCastModule.java

```diff
@@ -195,7 +195,7 @@ public void getStreamPosition(Promise promise) {
 
 
     @ReactMethod
-    public void startScan() {
+    public void startScan(@Nullable String appId) {
```

Comment:
I see `startDefaultScan` in the iOS implementation, but not in the android one.

Although, if you unify both functions into a single one (and this functions already looks like that!) it would be easier to maintain.